server/data/ciagApi/interfaces/ciagPlan.ts

```diff
@@ -31,7 +31,6 @@ export default interface CiagPlan {
   modifiedBy: string
   modifiedDateTime: string
   prisonId?: string
-  prisonName?: string
```

Comment:
This is the key property that we are removing, that has caused all the other change in this PR! 🤣